File: Views/src/components/stock-product/product/dtos/specificities-dto.ts
import { RootBase } from "src/shared/entities-dtos/root-base";
import { ManufacturerDto } from "./manufacturer-dto";
import { ProductDto } from "./product-dto";
import { ModelDto } from "./model-dto";

export class SpecificitiesDto extends RootBase {

  name: string;
  // speedLabel: string;
  // speed: string;
  // capacityLabel: string;
  // capacity: string;
  detailedDescription: string;
  // generation: string;
  description: string;
  manufacturerLink:string;
  products: ProductDto[];

}

File: Application/Services/Operations/StockProduct/Dtos/ProductKind/SpecificitiesDto.cs
using System.Collections.Generic;
using Domain.Entities.Shared;

namespace Application.Services.Operations.StockProduct.ProductKind
{
    public class SpecificitiesDto : RootBase
    {
        public string Name { get; set; }
        public string Capacity { get; set; }
        public string Speed { get; set; }
        public string Generation { get; set; }
        public string DetailedDescription { get; set; }
        public string Description { get; set; }
        public string ManufacturerLink { get; set; }
        public List<ProductDto> Products { get; set; }

    }
}